src/qt/test/wallettests.cpp

```diff
@@ -212,7 +212,8 @@ void TestGUI(interfaces::Node& node)
     // Check Request Payment button
     ReceiveCoinsDialog receiveCoinsDialog(platformStyle.get());
     receiveCoinsDialog.setModel(&walletModel);
-    RecentRequestsTableModel* requestTableModel = walletModel.getRecentRequestsTableModel();
+    auto requestTableModel = receiveCoinsDialog.recentRequestsTableModel();
```

Comment:
nanonit: diff could be smaller if `RecentRequestsTableModel*` remains.
Also using `auto` for simple types seems overkill ;)

Reply:
Done, previous version was unique_ptr.